### PR TITLE
Update WebXR guides for Markdown conversion

### PR DESCRIPTION
**Pull request by Elchi3**
Making content ready to convert to md.

### CODE REVIEW

**Review by AnilSeervi — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
I let you choose to follow up with AnilSeervi 's feedback in this bug or in a follow-up.

I'm approving so you can merge when it is the most appropriate for you.

The changes are all ok.

**Comment by AnilSeervi** [comment]
It's ok you can go ahead and merge this. I'll make a follow up PR with the changes.

File: files/en-us/web/api/webxr_device_api/inputs/index.html [hunk 1]
```diff
@@ -105,13 +105,13 @@ <h4 id="Grip_space">Grip space</h4>
 
 <strong>Figure: The coordinate system for the left hand's grip space.</strong>
 
-<img width="600px" alt="A diagram showing how the grip space indicates the local coordinate system for the player's hand relative to the world." src="dark_left.svg">
+<img alt="A diagram showing how the grip space indicates the local coordinate system for the player's hand relative to the world." src="dark_left.svg">
```
[the remark this answers is not in the document]
Comment (sideshowbarker) [reply files/en-us/web/api/webxr_device_api/inputs/index.html:110]: > The macro `XRInputSourceChangeEvent` on this page should be changed to `XRInputSourcesChangeEvent` with **Sources** in the macro.

That’s a good catch but I don’t think that needs to be fixed in this PR — that error seems to have existed in the docs from the very beginning, when the WebXR articles were first added.

**Comment by sideshowbarker** [comment]
> It's ok you can go ahead and merge this. I'll make a follow up PR with the changes.

I notice there are some other flaws reported:

* /en-US/docs/Web/API/XRInputSession does not exist
* /en-US/docs/Web/API/XRInputSession/targetRaySpace does not exist
* /en-US/docs/Web/API/GamepadMappingType does not exist
* Can't resolve /en-US/docs/Web/WebXR_Device_API/Gamepads
* Can't resolve /en-US/docs/Web/API/WebXR_Device_API/Input_profiles
* Can't resolve /en-US/docs/Web/API/WebXR_Device_API/Input_profiles
* Can't resolve /en-US/docs/Web/WebXR_Device_API/Gamepads

So if you’ll be doing a PR to fix the XRInputSourceChangeEvent flaw, you might also want to consider including changes for any of those other flaws which fixable.

**Comment by AnilSeervi** [comment]
> > It's ok you can go ahead and merge this. I'll make a follow up PR with the changes.
> 
> I notice there are some other flaws reported:
> 
> * /en-US/docs/Web/API/XRInputSession does not exist
> * /en-US/docs/Web/API/XRInputSession/targetRaySpace does not exist
> * /en-US/docs/Web/API/GamepadMappingType does not exist
> * Can't resolve /en-US/docs/Web/WebXR_Device_API/Gamepads
> * Can't resolve /en-US/docs/Web/API/WebXR_Device_API/Input_profiles
> * Can't resolve /en-US/docs/Web/API/WebXR_Device_API/Input_profiles
> * Can't resolve /en-US/docs/Web/WebXR_Device_API/Gamepads
> 
> So if you’ll be doing a PR to fix the XRInputSourceChangeEvent flaw, you might also want to consider including changes for any of those other flaws which fixable.

Yes, will see which ones are fixable.

**Pull request merged**